public/docs/ts/latest/guide/user-input.jade

```diff
@@ -87,7 +87,7 @@ figure.image-display
 
   Angular has a syntax feature called [**template reference variables**](./template-syntax.html#ref-vars).
   These variables grant us direct access to an element.
-  We declare a template reference variable by preceding an identifier with a hash/pound character (#).
+  We declare a template reference variable by preceding an identifier with the `let` keyword.
```

Comment:
(Yeah, my bad for just searching for (#) and not looking at the context carefully enough before editing.) Change reverted.